test.cc

```diff
@@ -1,24 +1,26 @@
 #include "product.h"
```

Comment:
Again, please do not make any changes to test.cc as it can change the public interface.